lib/turn.rb

```diff
@@ -0,0 +1,24 @@
+class Turn
+  attr_reader :guess, :card, :correct, :feedback
```

Comment:
We only need an `attr_reader` for our instance variables in order to be able to read the value of them. `correct` and `feedback` are defined on lines 9 and 17 so the `attr_reader` doesn't give us any benefit and therefore we don't need to include it for those two methods.

```suggestion
  attr_reader :guess, :card
```